### DISCUSSION TITLE
Clean Sync CarbonRom

### DISCUSSION THREAD

**Discussion by JV007xp** [ROM-builders/temporary#22788] [General]
please i need a clean sync 🙏🙏
https://cirrus-ci.com/task/6014718666604544?logs=build#L2

**Comment by Apon77** [comment]
your scripts are not buildable even in clean 
![image](https://github.com/ROM-builders/temporary/assets/61594531/0ea87cb7-eb55-4257-8366-9c9766605172)

**Reply by JV007xp** [reply]
This build is not from now, I don't understand

**Reply by Apon77** [reply]
![image](https://github.com/ROM-builders/temporary/assets/61594531/f29d9ad3-287d-40aa-846c-ad8453cd92b3)
